common/views/components/Divider/Divider.tsx

```diff
@@ -12,14 +12,15 @@ const Rule = styled.hr<Props>`
   border: 0;
   height: 1px;
   ${props =>
-    props.color && `background-color: ${props.theme.color(props.color)};`};
+    props.borderColor &&
+    `background-color: ${props.theme.color(props.borderColor)};`};
```

Comment:
Having a prop called `borderColor` that sets a `background-color` property seems liable to cause confusion.

Maybe `dividerColor`, like `dotColor`?

Reply:
Fair enough yeah 👍 might do `lineColor` so it reads `<Divider lineColor="yellow" />`